src/Providers/Trustpilot/Request/AccessTokenRequest.php

```diff
@@ -36,14 +36,23 @@ class AccessTokenRequest extends BaseRequest
      */
     const PASSWORD = 'password';
 
+    /**
+     * The sendRequest withAuthentication parameter must be set to false,
+     * this class is used AccessToken provider if not set to false it will
+     * go into infinite loop.
+     *
+     * @var bool
+     */
+    protected $withAuthentication = false;
+
     public function requiredFields(): array
     {
         return [
-            self::CLIENT_ID,
-            self::CLIENT_SECRET,
-            self::USERNAME,
-            self::PASSWORD,
-        ];
+                self::CLIENT_ID,
+                self::CLIENT_SECRET,
+                self::USERNAME,
+                self::PASSWORD,
+               ];
```

Comment:
this indentation looks wrong

Reply:
its the phpcs autofix default indentation. Will revert.